tss-esapi-sys/build.rs

```diff
@@ -372,6 +569,16 @@ pub mod tpm2_tss {
             )
         }
 
+        /// Use the pkg config file for a bundled installation
+        #[cfg(feature = "bundled")]
+        fn pkg_config(&self) {
+            pkg_config::Config::new()
+                .atleast_version(MINIMUM_VERSION)
+                .statik(true)
+                .probe(&self.name)
+                .expect(&format!("Failed to run pkg-config on {}", self.name));
```

Comment:
Same thing here. Change this to `unwrap_or_else(|_| panic!("Failed to run pkg-config on {}", self.name))`